README.md

```diff
@@ -23,37 +23,52 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
 
-## Running Stratum
-
-[Broadcom](stratum/hal/bin/bcm/standalone/README.md)
-
-[Tofino](stratum/hal/bin/barefoot/README.md)
-
-[bmv2](stratum/hal/bin/bmv2/README.md)
-
-[dummy](stratum/hal/bin/dummy/README.md)
-
-[NP4 Intel](stratum/hal/bin/np4intel/docker/README.md)
-
-## Other Topics
-
-[p4c-fpm](stratum/p4c_backends/README.md)
-
-[PHAL](stratum/docs/phal.md)
-
-[PHAL Development Tutorial](stratum/docs/phal_development_tutorial.md)
-
-[PHAL DB Configurators](stratum/docs/configurators.md)
-
-[gNMI](stratum/docs/gnmi/README.md)
-
-[cdlang](stratum/testing/cdlang/README.md)
-
-[testvectors](stratum/public/proto/tv.proto)
-
-[SSL/TLS support](stratum/lib/security/README.md)
-
-[TAI](stratum/docs/tai/README.md)
+## Table of Contents
+### Development
+
+- [Code of Conduct](/CODE_OF_CONDUCT.md)
+- [Contribution Guidelines](/CONTRIBUTING.md)
+- [IDE Setup](https://github.com/stratum/stratum/wiki/IDE-setup-for-development)
+- [PHAL Development Tutorial](/stratum/docs/phal_development_tutorial.md)
+- [Stratum Configurators Design Document](/stratum/docs/configurators.md)
+- [Stratum PHAL Overview (Physical Hardware Access Layer)](/stratum/docs/phal.md)
+
+### Platforms
+
+- [Tofino](/stratum/hal/bin/barefoot/README.md)
+  - [Running Stratum on a Barefoot Tofino based switch](/stratum/hal/bin/barefoot/README.run.md)
+  - [Testing Stratum on a Barefoot Tofino based switch](/stratum/hal/bin/barefoot/README.test.md)
+  - [P4Runtime p4_device_config formats](/stratum/hal/bin/barefoot/README.pipeline.md)
+  - [Building Stratum for Barefoot Tofino based switches](/stratum/hal/bin/barefoot/README.build.md)
+- [Broadcom](/stratum/hal/bin/bcm/standalone/README.md)
+  - [How to Add a Dataplane Feature to Stratum (BCM)](/stratum/docs/extenting_stratum_bcm.md)
+- [Intel NP4](/stratum/hal/bin/np4intel/docker/README.md)
+- [bmv2](/stratum/hal/bin/bmv2/README.md)
+- [Dummy Switch](/stratum/hal/bin/dummy/README.md)
+
+### Tools
+
+- [gNMI tool](/stratum/tools/gnmi/README.md)
+- [Stratum-enabled Mininet](/tools/mininet/README.md)
+- [P4Runtime write request replay tool](/stratum/tools/stratum-replay/README.md)
+
+### Other Documentation
+
+- [Installation guide: ONOS + Stratum + Barefoot Tofino switch](/stratum/docs/setup_guide_barefoot_tofino_onos.md)
+- [Trellis+Stratum example](/tools/mininet/examples/trellis/README.md)
+- [Community Channels](https://github.com/stratum/stratum/wiki/Discuss)
+- [Talks](https://github.com/stratum/stratum/wiki/Talks)
+- [testvectors](stratum/public/proto/tv.proto)
```

Comment:
This is the protobuf, maybe use this?
[](https://github.com/stratum/testvectors/blob/master/docs/testvectors_overview.md)

Reply:
```suggestion
- [Test Vectors](https://github.com/stratum/testvectors/blob/master/docs/testvectors_overview.md)
```